bin/setup

```diff
@@ -3,6 +3,8 @@
 Dir.chdir(File.expand_path("../..", __FILE__)) do
   # This script is a starting point to setup your application.
   # Add necessary setup steps to this file:
+  puts "== Installing system dependencies =="
+  system "bundle install phantomjs"
```

Comment:
Do we need to tell Travis that we need `phantomjs` installed in our `.travis.yml`?